store/storetest/job_store.go

```diff
@@ -77,15 +74,11 @@ func testJobGetAllByType(t *testing.T, ss store.Store) {
 		defer ss.Job().Delete(job.Id)
 	}
 
-	if received, err := ss.Job().GetAllByType(jobType); err != nil {
-		t.Fatal(err)
-	} else if len(received) != 2 {
-		t.Fatal("received wrong number of jobs")
-	} else if received[0].Id != jobs[0].Id && received[1].Id != jobs[0].Id {
-		t.Fatal("should've received first jobs")
-	} else if received[0].Id != jobs[1].Id && received[1].Id != jobs[1].Id {
-		t.Fatal("should've received second jobs")
-	}
+	received, err := ss.Job().GetAllByType(jobType)
+	require.Nil(t, err)
+	require.Len(t, received, 2)
+	require.False(t, received[0].Id != jobs[0].Id && received[1].Id != jobs[0].Id, "should've received first jobs")
```

Comment:
I think we can replace this "not semantic assertion" with for example `require.Contains(t, []string{received[0].Id, received[1].Id}, jobs[0].Id)` and the equivalent one for `jobs[1].Id`

Reply:
Thanks for the suggestion! I changed it for an `ElementsMatch` comparison, that can check for both ids in one check